{"name": "test_excel[round:url(https:\\/\\/ss1.bdstatic.com\\/5eN1bjq8AAUYm2zgoY3K\\/r\\/www\\/cache\\/static\\/protocol\\/https\\/global\\/img\\/feedback_add_ph]", "status": "failed", "statusDetails": {"message": "AssertionError: assert b'round:url(h...edback_add_ph' == '<Response [200]>'\n  +b'round:url(https:\\\\/\\\\/ss1.bdstatic.com\\\\/5eN1bjq8AAUYm2zgoY3K\\\\/r\\\\/www\\\\/cache\\\\/static\\\\/protocol\\\\/https\\\\/global\\\\/img\\\\/feedback_add_ph'\n  -'<Response [200]>'", "trace": "data = b'round:url(https:\\\\/\\\\/ss1.bdstatic.com\\\\/5eN1bjq8AAUYm2zgoY3K\\\\/r\\\\/www\\\\/cache\\\\/static\\\\/protocol\\\\/https\\\\/global\\\\/img\\\\/feedback_add_ph'\n\n    @pytest.mark.parametrize('data', ExcelHandler.get_excel_data(1))\n    def test_excel(data):\n>       assert data == '<Response [200]>'\nE       AssertionError: assert b'round:url(h...edback_add_ph' == '<Response [200]>'\nE         +b'round:url(https:\\\\/\\\\/ss1.bdstatic.com\\\\/5eN1bjq8AAUYm2zgoY3K\\\\/r\\\\/www\\\\/cache\\\\/static\\\\/protocol\\\\/https\\\\/global\\\\/img\\\\/feedback_add_ph'\nE         -'<Response [200]>'\n\ntest.py:16: AssertionError"}, "parameters": [{"name": "data", "value": "<class 'bytes'>"}], "start": 1586663329516, "stop": 1586663329517, "uuid": "aa5a31c1-0586-4214-a5c9-19c29d5b686c", "historyId": "873220b08cde6ba47bfc48635be897f2", "testCaseId": "1a84b5fa1cd8fc91bc1e7f314e541701", "fullName": "allure.test#test_excel", "labels": [{"name": "parentSuite", "value": "allure"}, {"name": "suite", "value": "test"}, {"name": "host", "value": "localhost"}, {"name": "thread", "value": "42329-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "allure.test"}]}